### PR TITLE
Add more integration tests

### PR DESCRIPTION
**Pull request by jsliacan**
This PR contains new integration tests. This is wip and I'll be glad to hear what to add/replace/remove/rework. 

- interaction with oc command (in `oc.feature`)
- end-to-end story creating an app (in `story.feature`)

The `oc` command interaction is only basic as we'll adopt OpenShift extended tests and we do not want to duplicate much work. The story is a very simple health check of the cluster (if an app can be deployed then at least something has to be working :) ). 

Thanks for feedback!

### CODE REVIEW

File: test/integration/crcsuite/crcsuite.go [hunk 1]
```diff
@@ -70,10 +80,9 @@ func FeatureContext(s *godog.Suite) {
 	})
 
 	s.AfterSuite(func() {
-		err := DeleteCRC()
-		if err != nil {
-			fmt.Println(err)
-		}
+
+		ForceStopCRC()
```
Comment (gbraad) [inline test/integration/crcsuite/crcsuite.go]: 👍

**Comment by jsliacan** [comment]
retest this please

**Comment by jsliacan** [comment]
retest this please

**Comment by jsliacan** [comment]
retest this please

**Comment by jsliacan** [comment]
retest this please

**Comment by jsliacan** [comment]
retest this please

**Comment by jsliacan** [comment]
retest this please

**Comment by praveenkumar** [comment]
retest this please.

**Comment by jsliacan** [comment]
In favour of #371 and #354. Minor typos and refactoring from this PR will be submitted as a separate PR later.

**Pull request closed without merging**